### ISSUE TITLE
Implement Age 10 cards for Artifacts Expansion

### ISSUE THREAD

**Issue by ultimatefiend** [micahstairs/bga-innovation#162]
This issue will track the implementation of all Age 10 cards for the Artifacts expansion. Please update this comment before working on any of these cards (if the work is unblocked) so that we don't have duplicate work being done. We no longer need to file a separate issue for each card, since a consolidated list of the cards will keep us more organized.

  - [x] **205 - Rover Curiosity**
  - [x] **206 - Higgs Boson**
  - [x] **207 - Exxon Valdez**
  - [x] **208 - Maldives**
  - [x] **209 - Maastricht Treaty**
  - [x] **210 - Seikan Tunnel** 
  - [x] **211 - Dolly the Sheep**
  - [x] **212 - Where's Waldo?**
  - [x] **213 - DeLorean DMC-12**
  - [x] **214 - Twister**
